docs/raspberry-pi.md

```diff
@@ -0,0 +1,17 @@
+# Raspberry Pi
+
+The playbook support for Raspberry Pi is now in beta(ish). The problem is that, docker containers are not architecture independent, and most of them are not build for arm. Some roles have been updated, so they built the necessary image on the host. It needs more space, as some build tools need to be present (like Java, for mxisd).
+
+To use these modification there is a variable that needs to be switched to enable this functionality. Add this to your vars.yaml file:
+```
+matrix_raspberry_pi = true
```

Comment:
This is not true anymore.

In its place, we should probably introduce a new `matrix_container_images_self_build` variable in `roles/matrix-base/defaults/main.yml` (defaulting to `false`).

In `group_vars/matrix_servers`, we can wire it to all other `*_self_build` variables. Example:

```yaml
matrix_riot_web_self_build: "{{ matrix_container_images_self_build }}"
```